pkg/csi_driver/controller.go

```diff
@@ -1001,3 +1010,12 @@ func (s *controllerServer) DeleteSnapshot(ctx context.Context, req *csi.DeleteSn
 
 	return &csi.DeleteSnapshotResponse{}, nil
 }
+
+func parseNfsExportOptions(optionsString string) ([]*file.NfsExportOptions, error) {
```

Comment:
consider adding some simple unit tests for this function

Reply:
It's covered in the e2e `nfs-export-options` unit test, but adding specific ones for the subfunction is a good idea, since we can easily isolate the behavior here

Reply:
Added